[2025-04-20T00:00:03.785+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-20T00:00:03.814+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: iniciar_tabla_mysql.start_table manual__2025-04-20T00:00:01.912346+00:00 [queued]>
[2025-04-20T00:00:03.831+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: iniciar_tabla_mysql.start_table manual__2025-04-20T00:00:01.912346+00:00 [queued]>
[2025-04-20T00:00:03.831+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-04-20T00:00:03.854+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): start_table> on 2025-04-20 00:00:01.912346+00:00
[2025-04-20T00:00:03.869+0000] {standard_task_runner.py:63} INFO - Started process 214392 to run task
[2025-04-20T00:00:03.877+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'iniciar_tabla_mysql', 'start_table', 'manual__2025-04-20T00:00:01.912346+00:00', '--job-id', '755', '--raw', '--subdir', 'DAGS_FOLDER/mysql_start_table.py', '--cfg-path', '/tmp/tmp9xpsfvlz']
[2025-04-20T00:00:03.884+0000] {standard_task_runner.py:91} INFO - Job 755: Subtask start_table
[2025-04-20T00:00:03.971+0000] {task_command.py:426} INFO - Running <TaskInstance: iniciar_tabla_mysql.start_table manual__2025-04-20T00:00:01.912346+00:00 [running]> on host e1dfa1bb16b8
[2025-04-20T00:00:04.123+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='iniciar_tabla_mysql' AIRFLOW_CTX_TASK_ID='start_table' AIRFLOW_CTX_EXECUTION_DATE='2025-04-20T00:00:01.912346+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-20T00:00:01.912346+00:00'
[2025-04-20T00:00:04.124+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-20T00:00:04.161+0000] {base.py:84} INFO - Using connection ID 'mysql_***_conn' for task execution.
[2025-04-20T00:00:04.309+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-20T00:00:04.310+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-20T00:00:04.320+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=iniciar_tabla_mysql, task_id=start_table, execution_date=20250420T000001, start_date=20250420T000003, end_date=20250420T000004
[2025-04-20T00:00:04.376+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-20T00:00:04.416+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-20T00:00:04.417+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
